### ISSUE TITLE
FID & Loss diverge on CIFAR10?

### ISSUE THREAD

**Issue by tj-14** [tqch/ddpm-torch#14]
Hi,

I used the following command to train the model on CIFAR10
`python train.py --dataset cifar10 --num-gpus 4 --use-ema --eval --distributed --rigid-launch`

which I was able to achieve the same FID at epoch 2040 as in the performance table. 
However, when I keep training it longer, the loss decreases, but FID is worse (picture below). I'm wondering if this is expected or if you have experienced this behavior before?

![image](https://github.com/tqch/ddpm-torch/assets/2980451/72b8ba69-eb6d-40dd-939f-13d6bed95752)

Small details that I'm not sure if they matter is that 1) I change the number of epoch in the config file after training the first 2040 epochs, and 2) I resume the training process on a different machine/GPU.

Thank you for your time!

**Comment by tqch** [comment]
Hi, thank you for testing the code and reporting your result. I think the divergence between training loss and FID was due to overfitting to the dataset and it was expected after FID hit its optimal. Similar observations have been made by many others and myself (e.g., https://github.com/openai/improved-diffusion/issues/91#issue-1711545164)

**Comment by tj-14** [comment]
Thank you for the explanation again :)
